dashboard/app/helpers/users_helper.rb

```diff
@@ -246,13 +246,9 @@ def script_progress_for_users(users, script)
         level_for_progress = level.get_level_for_progress(user, script)
         ul = user_levels_by_level.try(:[], level_for_progress.id)
 
-        # For contained levels, progress is stored with the contained level id but feedback is stored with
-        # the level id, which is why we need a different level id to get feedback for contained levels.
-        level_id_for_feedback = level.contained_levels.any? ? level.id : level_for_progress.id
```

Comment:
I removed this conditional for contained_levels because we don't need to fetch the review state for contained level, does that seem reasonable?